akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -173,7 +173,7 @@ private[akka] final case class Grouped[T](n: Int) extends PushPullStage[T, immut
   override def onPull(ctx: Context[immutable.Seq[T]]): SyncDirective =
     if (ctx.isFinishing) {
       val elem = buf.result()
-      buf.clear() //FIXME null out the reference to the `buf`?
+      buf.clear()
```

Comment:
Ah true, finishing a stage sets the backing array slot to a FinishedOp. GC was not the primary intention for that feature though, but it is nice anyway :)